.travis.yml

```diff
@@ -43,7 +43,8 @@ install:
   - source ~/.cargo/env || true
 
 before_script:
-  - rustup component add rustfmt-preview
```

Comment:
Changing this is an opportunistic tweak. It turns out that we can only run `rustfmt` on nightly, so we may as well avoid installing it when we're on stable.